src/Numerics/Mesh/Interpolation.jl

```diff
@@ -225,9 +224,9 @@ struct InterpolationBrick{
 
         end # el loop
 
-        m_ξ1, _ = GaussQuadrature.legendre(FT, qm[1], GaussQuadrature.both)
-        m_ξ2, _ = GaussQuadrature.legendre(FT, qm[2], GaussQuadrature.both)
-        m_ξ3, _ = GaussQuadrature.legendre(FT, qm[3], GaussQuadrature.both)
+        m_ξ1, _ = Elements.gridpoints(FT, qm[1] - 1)
+        m_ξ2, _ = Elements.gridpoints(FT, qm[2] - 1)
+        m_ξ3, _ = Elements.gridpoints(FT, qm[3] - 1)
```

Comment:
I think you can use just `referencepoints(grid)` here so you don't actually need `gridpoints` function.

Reply:
Even better. Then, my comment above won't be needed